lib/utils.js

```diff
@@ -206,7 +206,13 @@ function getBowerContent(from) {
  * @param {requestCallback} callback - The callback that handles the response.
  */
 function getNpmRc(callback) {
-    let npmRcFilePath = path.join(os.homedir(), ".npmrc");
+    let npmRcFilePath;
+
+    if (process.env.NODE_ENV !== "test") {
+        npmRcFilePath = path.join(os.homedir(), ".npmrc");
+    } else {
+        npmRcFilePath = path.join(__dirname, "..", "test/data/.npmrc");
+    }
```

Comment:
If you would encapsulate that in an object, you could have a variable `npmrcfinder` that you could dependency inject with different piece of code in test and in production.